migrations/20181011181010_add_dishes_table.js

```diff
@@ -0,0 +1,15 @@
+
+exports.up = function (knex, Promise) {
+  return knex.schema.createTable('dishes', function (tbl) {
+    // we must use the callback syntax for .createTable()
+    tbl.increments('dish_id'); // pass the name if you wanted to be called anything other than id
```

Comment:
This isn't necessarily wrong, but it's pretty standard to just leave this field as `id`. You will be able to differentiate it from other tables with `id` because you will have to say `dishes.id` or `recipe.id`.